Bottom: e2131f7e51447fe3fd85c9a62239f937dac571b5
Top:    9bd029a43201d8d7586d4167218571a9c818e2d0
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-05-16 15:13:49 -0700

Refresh of rstream-spinpoll

---

diff --git a/examples/rstream.c b/examples/rstream.c
index 27a6aeb..75e5246 100644
--- a/examples/rstream.c
+++ b/examples/rstream.c
@@ -55,7 +55,6 @@ static int test_size[] = {
 
 static int use_rs = 1;
 static int use_async;
-static int poll_timeout = -1;
 static int verify;
 static int flags = MSG_DONTWAIT;
 static int no_delay = 1;
@@ -502,6 +501,7 @@ static int set_test_opt(char *optarg)
 			break;
 		case 'a':
 			use_async = 1;
+			flags |= MSG_DONTWAIT;
 			break;
 		case 'b':
 			flags &= ~MSG_DONTWAIT;
@@ -511,9 +511,6 @@ static int set_test_opt(char *optarg)
 			flags |= MSG_DONTWAIT;
 			no_delay = 1;
 			break;
-		case 'p':
-			poll_timeout = 0;
-			break;
 		case 'v':
 			verify = 1;
 			break;
@@ -531,8 +528,6 @@ static int set_test_opt(char *optarg)
 		} else if (!strncasecmp("nonblock", optarg, 8)) {
 			flags |= MSG_DONTWAIT;
 			no_delay = 1;
-		} else if (!strncasecmp("poll", optarg, 4)) {
-			poll_timeout = 0;
 		} else if (!strncasecmp("verify", optarg, 6)) {
 			verify = 1;
 		} else {
@@ -586,7 +581,6 @@ int main(int argc, char **argv)
 			printf("\t    a|async - asynchronous operation (use poll)\n");
 			printf("\t    b|blocking - use blocking calls\n");
 			printf("\t    n|nonblocking - use nonblocking calls\n");
-			printf("\t    p|poll - poll on asynchronous operations\n");
 			printf("\t    v|verify - verify data\n");
 			exit(1);
 		}
